topics/mvvmcross/index.md

```diff
@@ -0,0 +1,12 @@
+---
+aliases: monocross
+created_by: Alexandru Stefan Popescu
+display_name: MvvmCross
+github_url: https://github.com/mvvmcross
+logo: mvvmcross.png
+released: June 2013
+short_description: The .NET MVVM framework for cross-platform solutions, including Xamarin.iOS, Xamarin.Android, Windows and Mac.
+topic: mvvmcross
+url: https://www.mvvmcross.com/
+---
+The .NET MVVM framework for cross-platform solutions, including Xamarin.iOS, Xamarin.Android, Windows and Mac.
```

Comment:
Can you expand this to be more than the short_description? This can be several sentences long, to better explain what the topic is about.

Reply:
Done. :D